web-server/src/content/DoraMetrics/AIAnalysis/AIAnalysis.tsx

```diff
@@ -0,0 +1,306 @@
+                table: { m: -1 },
+                th: { color: theme.colors.primary.main, fontSize: '1.1em' },
+                'td:first-of-type': {
+                  fontWeight: 'bold',
+                  color: theme.colors.secondary.main
+                },
+                'th, td': { textAlign: 'left', verticalAlign: 'top', p: 1 }
+              }}
+            >
+              <TabContext value={selectedTab.value}>
+                <FlexBox alignCenter gap1 px={1} fullWidth>
+                  <IconButton
+                    size="small"
+                    onClick={() =>
+                      selectedTab.set((n) =>
+                        n === '0' ? '5' : String(Number(n) - 1)
+                      )
+                    }
+                  >
+                    <ChevronLeftRounded />
+                  </IconButton>
+                  <TabList
+                    onChange={(_, n) => selectedTab.set(n)}
+                    sx={{ mt: 1, flex: 1 }}
+                  >
+                    <Tab label="Summary" value="0" />
+                    <Tab label="Trends" value="1" />
+                    <Tab label="Lead Time" value="2" />
+                    <Tab label="Deployments" value="3" />
+                    <Tab label="Failure Rate" value="4" />
+                    <Tab label="Recovery Time" value="5" />
+                  </TabList>
+
+                  <IconButton
+                    size="small"
+                    onClick={() =>
+                      selectedTab.set((n) => String((Number(n) + 1) % 6))
+                    }
+                  >
+                    <ChevronRightRounded />
+                  </IconButton>
+                </FlexBox>
+                <TabPanel value="0">
+                  <Markdown>{data.dora_compiled_summary}</Markdown>
+                </TabPanel>
+                <TabPanel
+                  value="1"
+                  sx={{ 'table td:first-of-type': { whiteSpace: 'nowrap' } }}
+                >
+                  <Markdown>{data.dora_trend_summary}</Markdown>
+                </TabPanel>
+                <TabPanel
+                  value="2"
+                  sx={{ 'table td:first-of-type': { whiteSpace: 'nowrap' } }}
+                >
+                  <Markdown>{data.lead_time_trends_summary}</Markdown>
+                </TabPanel>
+                <TabPanel
+                  value="3"
+                  sx={{ 'table td:first-of-type': { whiteSpace: 'nowrap' } }}
+                >
+                  <Markdown>
+                    {data.deployment_frequency_trends_summary}
+                  </Markdown>
+                </TabPanel>
+                <TabPanel
+                  value="4"
+                  sx={{ 'table td:first-of-type': { whiteSpace: 'nowrap' } }}
+                >
+                  <Markdown>{data.change_failure_rate_trends_summary}</Markdown>
+                </TabPanel>
+                <TabPanel
+                  value="5"
```

Comment:
all these values can be combined into an enum:
``` typescript
enum MyEnum {
dora_trend_summary,
change_failure_rate_trends_summary,
mean_time_to_recovery_trends_summary
...more...
}
```
this means that:
`MyEnum.dora_trend_summary` will be equal to 0